### PR TITLE
Fix support form bugs, add missing pages

### LINKED ISSUE #305: Support: clean up support pages
- [x] Bring in missing corporate support page
- [x] Bring in missing success/error pages to the theme
- [x] Fix casing on template titles (should be sentence case)

### PR DESCRIPTION
**Pull request by agjohnson**
See #305 for screenshots.

- Fixes #305

### CODE REVIEW

**Comment by agjohnson** [comment]
I need this to hotfix so will merge. @humitos please review this after still.

File: readthedocsext/theme/templates/support/index.html [hunk 1]
```diff
@@ -4,55 +4,66 @@
...
+{% block title %}{% trans "Support request" %}{% endblock %}
 
 {% block content %}
-  <div id="support_page">
+  <h1 class="ui large heading">{% trans "Support request" %}</h1>
 
-    <h2 id="usage-questions">Usage Questions</h2>
+  {% block support_form %}
+    {# TODO this is a check for our commercial hosting, we should do something different here #}
+    {% if USE_ORGANIZATIONS %}
```
Comment (humitos) [inline readthedocsext/theme/templates/support/index.html:14]: We have been using `ALLOW_PRIVATE_REPOSITORIES` to differentiate between org/com.

However, I think we should have a specific setting for this and stop using other settings that were meant for other things.

In this particular case, when we support organizations on org --we will need to change all these settings.

File: readthedocsext/theme/templates/support/index.html [hunk 2]
```diff
@@ -4,55 +4,66 @@
...
+      </p>
+    {% else %} 
+      <h2 id="usage-questions">Usage questions</h2>
 
-    <p>
-      If you have questions about how to use Read the Docs, or have an issue that isn't related to a bug, <a href="http://stackoverflow.com/questions/tagged/read-the-docs">Stack Overflow</a> is the best place to ask. Tag questions with <span class="pre">read-the-docs</span> so other folks can find them easily.
-    </p>
+      <p>
+        If you have questions about how to use Read the Docs, or have an issue that isn't related to a bug, <a href="http://stackoverflow.com/questions/tagged/read-the-docs" target="_blank">Stack Overflow</a> is the best place to ask. Tag questions with <span class="pre">read-the-docs</span> so other folks can find them easily.
+      </p>
```
Comment (humitos) [inline readthedocsext/theme/templates/support/index.html:29]: Do we want to translate this whole page?

**Review by humitos — COMMENTED** [review]
Looks good 👍🏼

File: readthedocsext/theme/templates/support/index.html (hunk 1 above)
Comment (agjohnson) [reply readthedocsext/theme/templates/support/index.html:14]: I've been using `USE_ORGANIZATIONS`, agreed we should have a specific setting though.

File: readthedocsext/theme/templates/support/index.html (hunk 2 above)
Comment (agjohnson) [reply readthedocsext/theme/templates/support/index.html:29]: We should yeah. I dropped some small fixes in for localization but didn't want to play with this too much in this bug fix.

**Pull request merged**